### PR TITLE
[CURA-9005] Restyle print setting category

### PR DESCRIPTION
**Pull request by Joeydelarago**
See [https://github.com/Ultimaker/Cura/pull/11622](https://github.com/Ultimaker/Cura/pull/11622) for more information

### CODE REVIEW

File: UM/Qt/qml/UM/CategoryButton.qml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+    property var expanded: false
+
+    property alias arrow: categoryArrow
+    property alias categoryIcon: icon.source
+    property alias labelText: categoryLabel.text
+
+    property color text_color:
+    {
+        if (!base.enabled)
```
Comment (nallath) [inline UM/Qt/qml/UM/CategoryButton.qml:23]: Shouldn't this also be handled in the states?

File: UM/Qt/qml/UM/CategoryButton.qml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                PropertyChanges { target: backgroundRectangle; color: UM.Theme.getColor("setting_category_hover")}
+            }
+        ]
+    }
+
+    contentItem: Item
+    {
+        anchors.fill: parent
+
+        Label
```
Comment (nallath) [inline UM/Qt/qml/UM/CategoryButton.qml:73]:
```suggestion
        UM.Label
```

**Review by nallath — CHANGES_REQUESTED** [review]

File: UM/Qt/qml/UM/CategoryButton.qml (hunk 1 above)
Comment (casperlamboo) [reply UM/Qt/qml/UM/CategoryButton.qml]: https://github.com/Ultimaker/Uranium/pull/794/commits/a0faefcab0bc85706975cd7d429cded2e30cfc63

File: UM/Qt/qml/UM/CategoryButton.qml [hunk 3]
```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2022 Ultimaker B.V.
+// Uranium is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.2
+import QtQuick.Controls 2.1
+
+import UM 1.5 as UM
+
+Button
```
[the remark this answers is not in the document]
Comment (casperlamboo) [reply UM/Qt/qml/UM/CategoryButton.qml:16]: https://github.com/Ultimaker/Uranium/pull/794/commits/e0cfe8c6420b43d59846b86b1265406cb3bbed59

**Review by nallath — APPROVED** [review]

**Pull request merged**